registry.json

```diff
@@ -4221,5 +4221,20 @@
     "telegram": "saganpool",
     "operatorName": "John",
     "github": ""
+  },
+  {
+    "ticker": "CBITS",
+    "poolId": "6782865c91a230fb379a433a0630a68a9adcdca9e518ad73302ef805",
+    "social": {
+      "website": "https://cardanobits-pool.com/",
+      "twitter": "MavidDuster",
+      "discord": "MavidDuster#8325",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "https://t.me/cardanobitsofficial",
```

Comment:
Please can you change to the telegram of an individual and not a user or group? 

Also please remove the link - this should just be the handle, e.g. `MavidDuster`

Reply:
Changes were made and pull request was handed in